app/controllers/concerns/rendering.rb

```diff
@@ -50,5 +51,14 @@ def serializer
     end
 
     def includes; end
+
+    def pagination
+      {
+        limit:
+          params[:include_limit].present? ? params[:include_limit].to_i : 10,
```

Comment:
Just a note, if we set a default limit for rules, we'll need introduce server-side paginating in the frontend, because currently it requests/expects all rules.

Reply:
We maybe also want an upper limit here :thinking: :question:

Reply:
I'd rather figure out a way to determine a query's complexity and return some 4XX if the DB complexity is beyond some limit. So, you can request all rules, perhaps, but if you also want to include all profiles, too, it'd be potentially too complex. I'm not sure if this is really feasible, though. Still, if you request all benchmarks and include rules, it's possible to have included something like 12k rules, which would require a very long response time and something we want to avoid servicing.